source/models/prediction.py

```diff
@@ -28,13 +29,11 @@ def value_check(feature_dict):
     Args:
         feature_dict (dict) dictionary of the features,
             derived from the cfg dictionary object.
-
     """
     edited_input = []
-    round_nr = 0
     while len(edited_input) != len(feature_dict):
-        for key, value in feature_dict.items():
-            print(f"Write the data you want to be predicted {round_nr+1}/4:")
+        for count, (key, value) in enumerate(feature_dict.items(), 1):
+            print(f"Write the data you want to be predicted {count}/4:")
```

Comment:
Why do you write "4"?

Reply:
has been updated with: {len(feature_dict)}